content/src/logic/deployments.ts

```diff
@@ -8,13 +8,24 @@ import { AppComponents } from '../types'
 import { deploymentExists } from './database-queries/deployments-queries'
 
 export async function isEntityDeployed(
-  components: Pick<AppComponents, 'deployedEntitiesFilter' | 'database'>,
+  components: Pick<AppComponents, 'deployedEntitiesFilter' | 'database' | 'metrics'>,
   entityId: string
-) {
+): Promise<boolean> {
   // this condition should be carefully handled:
   // 1) it first uses the bloom filter to know wheter or not an entity may exist or definitely don't exist (.check)
   // 2) then it checks against the DB (deploymentExists)
-  return (await components.deployedEntitiesFilter.check(entityId)) && (await deploymentExists(components, entityId))
+  if (await components.deployedEntitiesFilter.check(entityId)) {
+    if (await deploymentExists(components, entityId)) {
+      components.metrics.increment('dcl_deployed_entities_filter_checks_total', { false_positive: 'no' })
```

Comment:
can `false_positive` become `bloom_hit` to prevent double negation?

Reply:
yes, it felt weird. `hit` alone would be good? prefer not to expose which kind of filter it is

Reply:
From a devexp point of view, the fact that it is a bloom filter tells you a lot about the implementation and the code you are reading, if filter is the name of the component and bloom filter isn't, somebody may think that it is simply a filter that returns true and false with 100% accuracy, which bloom filters don't do.

Reply:
Hmmm that's a good point... Now i want to add that to the metric and the variable:
`deployedEntitiesFilter` -> `deployedEntitiesBloomFilter`
`dcl_deployed_entities_filter_checks_total` -> `dcl_deployed_entities_bloom_filter_checks_total`